BaseFilter/sections/specific.txt

```diff
@@ -18281,6 +18280,9 @@ circleofcricket.com#$##myModal { display: none !important; }
 circleofcricket.com##div.container-full.add:has(> a[href]:first-child)
 ||circleofcricket.com/assets/adv/
 !
+! Hearst Stations Inc
```

Comment:
In their footer - `Hearst Television Inc`

```suggestion
! Hearst Television Inc
```

Reply:
I had not made up my mind about fixing that, as I was thinking about merging their news stations too, such as https://www.chron.com/ and https://www.theintelligencer.com/.

Reply:
Should I change it to Hearst Television Inc and leave it as is, or should I merge the news sites into it and change the name to Hearst Communications Inc?